tools/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "require": {
+        "doctrine/coding-standard": "^12.0.0",
+        "jetbrains/phpstorm-stubs": "2023.3",
+        "nikic/php-parser": "^4.18.0",
+        "phpbench/phpbench": "^1.1.1",
+        "phpstan/phpstan": "^1.10.60",
+        "phpstan/phpstan-phpunit": "^1.3.16",
+        "phpunit/phpunit": "^10.5.12",
+        "vimeo/psalm": "^5.23.0",
+        "roave/backward-compatibility-check": "^8.6.0",
+        "roave/infection-static-analysis-plugin": "^1.35.0"
+    },
+    "config": {
+        "allow-plugins": {
+            "infection/extension-installer": true,
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        },
+        "platform": {
+            "php": "8.1.99"
+        }
+    },
+    "autoload": {
```

Comment:
Thinking of this further, we should probably add the parent package as a dependency, not as an autoload definition: that makes the parent package able to declare its own dependencies, and the tooling follows only